lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -127,17 +127,18 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
       LocationStack;
 
 #ifndef NDEBUG
-  using UUSTuple = std::pair<std::pair<unsigned, unsigned>, StringRef>;
+  using UUSTuple = std::pair<unsigned, StringRef>;
```

Comment:
I suppose the old name meant unsigned-unsigned-string tuple; now the name doesn't make sense any more. Is there a better name for it?

Reply:
maybe just don't use the using decl at all